mifospay/src/main/java/org/mifos/mobilewallet/mifospay/auth/ui/LoginActivity.java

```diff
@@ -67,6 +67,7 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         getActivityComponent().inject(this);
         setContentView(R.layout.activity_login);
```

Comment:
@Bhavnaharitsa `activity_login` layout file won't be required after using the `SplashTheme`. You can remove this line and delete the layout file